### ISSUE TITLE
Use WAVE Accessibility Extension to check for issues with contrast and styling

### ISSUE THREAD

**Issue by canaleal** [Lynx-Data-Processing/Svelte-City-Twin-Mapping-Dashboard#5]
The WAVE Accessibility Extension shows issues with accessibility that lighthouse is unable to process. 
Use this extension to find any issues with the UI

https://addons.mozilla.org/en-CA/firefox/addon/wave-accessibility-tool/
